assets/js/components/indicators.js

```diff
@@ -59,6 +59,13 @@ export class IndicatorSection {
     this.comparisonMenu.$element.on('option-select', ((e) => {
       this.updateChartComparison(e.detail.option);
     }));
+
+    this.$element.on('click-col', ((e) => {
+      const containerId = $(e.currentTarget).attr('class');
+      const buttonId = $(e.target).attr('colid');
```

Comment:
you could just pass colId in `CustomEvent(name, {detail: {colId: colId}})`   then you don't have to make assumptions about the attributes of the elements - the interface between the chart and the page JS is then more and more just the event.